packages/admin-next/dashboard/src/lib/client/client.ts

```diff
@@ -57,3 +57,10 @@ export const client = {
   stockLocations: stockLocations,
   workflowExecutions: workflowExecutions,
 }
+
+export const sdk = new Medusa({
+  baseUrl: MEDUSA_BACKEND_URL,
```

Comment:
replace with `__BACKEND_URL__ || "http://localhost:9000"`, since we are deleting the `medusa` file once we have removed all v1 code.